tests/TestCase/Network/ResponseTest.php

```diff
@@ -1120,26 +1155,26 @@ public function corsData()
-            [$secureRequest, 'https://www.bar.com', 'http://www.bar.com', '', '', false, false],
-            [$secureRequest, 'https://www.bar.com', '*.bar.com', '', '', 'https://www.bar.com', false],
-
-            [$fooRequest, 'http://www.foo.com', '*', 'GET', '', '*', 'GET'],
-            [$fooRequest, 'http://www.foo.com', '*.foo.com', 'GET', '', 'http://www.foo.com', 'GET'],
-            [$fooRequest, 'http://www.foo.com', '*.foo.com', ['GET', 'POST'], '', 'http://www.foo.com', 'GET, POST'],
-
-            [$fooRequest, 'http://www.foo.com', '*', '', 'X-CakePHP', '*', false, 'X-CakePHP'],
-            [$fooRequest, 'http://www.foo.com', '*', '', ['X-CakePHP', 'X-MyApp'], '*', false, 'X-CakePHP, X-MyApp'],
-            [$fooRequest, 'http://www.foo.com', '*', ['GET', 'OPTIONS'], ['X-CakePHP', 'X-MyApp'], '*', 'GET, OPTIONS', 'X-CakePHP, X-MyApp'],
+            // [$fooRequest, null, '*', '', '', false, false, false, false, false],
+            // [$fooRequest, 'http://www.foo.com', '*', '', '', false, false, false, '*', false],
+            // [$fooRequest, 'http://www.foo.com', 'www.foo.com', '', '', false, false, false, 'http://www.foo.com', false],
+            // [$fooRequest, 'http://www.foo.com', '*.foo.com', '', '', false, false, false, 'http://www.foo.com', false],
+            // [$fooRequest, 'http://www.foo.com', 'http://*.foo.com', '', '', false, false, false, 'http://www.foo.com', false],
+            // [$fooRequest, 'http://www.foo.com', 'https://www.foo.com', '', '', false, false, false, false, false],
+            // [$fooRequest, 'http://www.foo.com', 'https://*.foo.com', '', '', false, false, false, false, false],
+            // [$fooRequest, 'http://www.foo.com', ['*.bar.com', '*.foo.com'], '', '', false, false, false, 'http://www.foo.com', false],
+
+            // [$secureRequest, 'https://www.bar.com', 'www.bar.com', '', '', false, false, false, 'https://www.bar.com', false],
+            // [$secureRequest, 'https://www.bar.com', 'http://www.bar.com', '', '', false, false, false, false, false],
+            // [$secureRequest, 'https://www.bar.com', '*.bar.com', '', '', false, false, false, 'https://www.bar.com', false],
+
+            // [$fooRequest, 'http://www.foo.com', '*', 'GET', '', false, false, false, '*', 'GET'],
+            // [$fooRequest, 'http://www.foo.com', '*.foo.com', 'GET', '', false, false, false, 'http://www.foo.com', 'GET'],
+            // [$fooRequest, 'http://www.foo.com', '*.foo.com', ['GET', 'POST'], '', false, false, false, 'http://www.foo.com', 'GET, POST'],
+            // [$fooRequest, 'http://www.foo.com', '*', '', 'X-CakePHP', false, false, false, '*', false, 'X-CakePHP'],
+            // [$fooRequest, 'http://www.foo.com', '*', '', ['X-CakePHP', 'X-MyApp'], false, false, false, '*', false, 'X-CakePHP, X-MyApp'],
+            // [$fooRequest, 'http://www.foo.com', '*', ['GET', 'OPTIONS'], ['X-CakePHP', 'X-MyApp'], false, false, false, '*', 'GET, OPTIONS', 'X-CakePHP, X-MyApp'],
```

Comment:
I was adding a set for the new options (the options for that test are cumbersome), and meant to uncomment those when my new one passed.